### ISSUE TITLE
[A11y] Line chart with a single data series does not announce series name, series contains undefined aria-label attribute

### ISSUE THREAD

**Issue by umkoksal** [highcharts/highcharts#16884]
#### Expected behaviour
When there are multiple line series in a line chart, each line announces itself via an aria-label when the user is traversing through that line. We expect the behavior to be the same when there is only a single line series in the line chart as well.

#### Actual behaviour
When there is only a single line series, the 'name' for that line series is shown in the tooltip, but it is never announced through the screen reader because the tracker for the series contains an undefined aria-label, whereas this aria-label is defined when there are multiple line series.

#### Live demo with steps to reproduce
Below is the jsfiddle where I have removed every line but 1 from the example provided in the 'Accessible line chart' in HighCharts documentation.

Please observe that the screen readers do not give the user any clue about the name of the data series when this is the case.
https://jsfiddle.net/h8c3Lkwy/

This is the HTML element in question, which displays an undefined aria-label
![image](https://user-images.githubusercontent.com/61870580/149504589-6c5f63fb-42e8-4ec9-aab4-636d89ebc892.png)

And yet the same aria-label is defined when I add another data series to the same chart:
![image](https://user-images.githubusercontent.com/61870580/149504868-ba29f32d-b937-4e7e-8626-216f314b2218.png)

#### Product version
v9.3.2

#### Affected browser(s)
Tested on Chromium Edge Version 97.0.1072.55

Since issue is with an empty aria-label, it can be assumed that all browsers who operate within aria-label defined accessible UIs are affected as well.

We would like to announce the name of the data series to our users, even if it is a single series (this is crucial in cases where the data point conveys additional information other than the chart title). Please let us know if this is the intended behavior and/or advise if there's a way to replicate our desired behavior through some chart config changes.

**Comment by oysteinmoseng** [comment]
Hi @umkoksal, this is actually intended behavior, to reduce verbosity for simple charts, but we are considering changing the defaults. For now, to override this behavior you can set [accessibility.series.describeSingleSeries](https://api.highcharts.com/highcharts/accessibility.series.describeSingleSeries) to `true`.

Example: https://jsfiddle.net/5s1Lzh2c/

**Comment by oysteinmoseng** [comment]
Defaults now changed to expose the series name per point if the `series.name` option is defined.

We are still not exposing the whole series group, to avoid landmark bloat by default for 1-series charts (note the `role="region"` on the `<g>` element in your post). So if this is desired and you want the full description on the `<g>` element, you'll still want to use the `describeSingleSeries` option mentioned above.

Also note that the formats of these descriptions can be configured with API options - see [Advanced accessibility configuration](https://www.highcharts.com/docs/accessibility/configure-the-accessibility-module).

**Comment by umkoksal** [comment]
@oysteinmoseng Thanks a lot for the swift response. I agree that the intended behavior is perfectly fine as well but we were requested by multiple users on our side. We are also very grateful for the workaround you have pointed out which has worked for us.
